packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -239,50 +239,6 @@ describeNoCompat("Summaries", (getTestObjectProvider) => {
             "Expected .attributes blob in default root DDS summary tree.");
     });
 
-    it("Should generate summary tree with isolated channels disabled", async () => {
```

Comment:
Do we have tests around reading from summary that was written with disableIsolatedChannels? That scenario is still supported so it will great to have tests around it.

Reply:
Not sure if we have any tests. Should we file a bug to track it ?